README.md

```diff
@@ -1 +1,22 @@
-# RMF simulator plugins
+# rmf\_simulation
+
+This repository contains simulation plugins used in Open-RMF. It currently supports Gazebo Classic 11 and Gazebo Fortress.
+
+Simulation plugins are split into two categories:
+
+1. Building simulation plugins under `rmf_building_sim_*`, are used to simulate different aspect of buildings such as doors, lifts and crowds (currently through the [Menge library](https://github.com/open-rmf/menge_vendor)). Utility plugins to help with simulation, such as UI widgets to toggle battery consumption and set charging behavior, as well as toggling floor visibility, are also placed in this category.
```

Comment:
Would be a bit nicer if each plugin name had its own indented bullet with an explanation. Also good if we can highlight (in a table?) the name of each plugin, whether its a world or model plugin, and the SDF schema when including this plugin eg door 
```
      <plugin name="door" filename="libdoor.so">
        <v_max_door>0.5</v_max_door>
        <a_max_door>0.3</a_max_door>
        <a_nom_door>0.15</a_nom_door>
        <dx_min_door>0.01</dx_min_door>
        <f_max_door>500.0</f_max_door>
        <door name="hardware_door" type="SwingDoor" left_joint_name="empty_joint" right_joint_name="right_joint" />
      </plugin>
```

Bonus if we can document each parameter in the plugins